infrastructure/docker/build/Dockerfile-traffic_portal

```diff
@@ -35,12 +37,20 @@ RUN	yum -y install \
 		libffi-devel \
 		make \
 		nodejs \
-		npm \
-		ruby-devel \
-		rubygems
+		npm
+
+# NOTE: error installing with default ruby 2.0.0.   This allows us to upgrade to 2.3
+RUN	yum -y install \
+		centos-release-scl \
+		openssl-devel && \
+	yum -y install \
+		rh-ruby23 \
+		rh-ruby23-ruby-devel \
+		rh-ruby23-runygems
```

Comment:
runygems is probably not right :)

Reply:
just checked -- rubygems gets installed anyway because another package depends on it.  Turns out that `yum install ...` succeeds if one of the packages to be installed succeeds,  so no error here..